chia/wallet/puzzles/dao_proposal.clsp

```diff
@@ -308,7 +314,8 @@
                     )
                     ; Passed
                     (list
-                      (list CREATE_PUZZLE_ANNOUNCEMENT (sha256tree (list PROPOSED_PUZ_HASH 0)))  ; the 0 at the end is announcement_args in proposal_validators
+                      ; (list CREATE_PUZZLE_ANNOUNCEMENT (sha256tree (list PROPOSED_PUZ_HASH 0)))  ; the 0 at the end is announcement_args in proposal_validators
```

Comment:
```suggestion
                      ; (list CREATE_PUZZLE_ANNOUNCEMENT (sha256tree (list PROPOSED_PUZ_HASH 0)))  ; the 0 at the end is announcement_args in proposal_validators
```
; We use a coin announcement instead of a puzzle announcement because we want the validator to match this particular step in the proposal's lifetime, and ... ... ...
; the 0 at the end is announcement_args in proposal_validators